src/components/Book.js

```diff
@@ -3,12 +3,43 @@ import PropTypes from 'prop-types';
 
 const Book = ({ book, handleRemoveBook }) => (
   <>
-    <tr>
-      <td>{book.bookId}</td>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="submit" onClick={() => handleRemoveBook(book)}>Remove Book</button></td>
-    </tr>
+    <div className="Lesson-Panel">
+      <div className="Lesson-Panel-left">
+        <div className="category">{book.category}</div>
+        <div className="title">{book.title}</div>
+        <div className="author">Author Name</div>
+        <div className="links">
+          <div className="link comments">Comments</div>
+          <div className="link remove">
+            <input
+              className="Remove"
+              type="submit"
+              onClick={() => handleRemoveBook(book)}
+              value="Remove"
+            />
+          </div>
+          <div className="link edit">Edit</div>
+        </div>
+      </div>
+
+      <div className="Lesson-Panel-middle">
+        <div className="progress-circle">
+          <div className="circle" />
+        </div>
+        <div className="progress-info">
+          <div className="progress-info-percentage">64%</div>
+          <div className="progress-info-status">Completed</div>
+        </div>
```

Comment:
### [OPTIONAL]

- [ ] This progress score can be a random number between 1 - 100 generated from Math.random()